From b5d35bbf42e177d60b93d90863c06fac5ac83000 Mon Sep 17 00:00:00 2001
From: Hideki Yamane <henrich@debian.org>
Date: Tue, 2 Jun 2020 23:27:00 +0900
Subject: workaround for libwoff*-dev

During fontforge built, found devlibs error as below

Library package automatic movement utility
 --> libfreetype6-dev package exists.
 --> libgif-dev package exists.
 --> libglib2.0-dev package exists.
 --> libjpeg-dev package exists.
 --> libpng-dev package exists.
 --> libpython3.8-dev package exists.
 --> libreadline-dev package exists.
 --> libspiro-dev package exists.
 --> libtiff5-dev package exists.
 --> libuninameslist-dev package exists.
devlibs error: There is no package matching [libwoff2dec1.0.2-dev] and noone provides it, please report bug to d-shlibs maintainer
devlibs error: There is no package matching [libwoff2enc1.0.2-dev] and noone provides it, please report bug to d-shlibs maintainer
 --> libxml2-dev package exists.
 --> zlib1g-dev package exists.
---
 d-devlibdeps | 1 +
 1 file changed, 1 insertion(+)

diff --git a/d-devlibdeps b/d-devlibdeps
index aef6bfa..e8e48f8 100755
--- a/d-devlibdeps
+++ b/d-devlibdeps
@@ -197,6 +197,7 @@ overridedevlibdeps() {
 		-e 's/libvo-amrwbenc0-dev/libvo-amrwbenc-dev/' \
 		-e 's/libvorbis0-dev/libvorbis-dev/' \
 		-e 's/libvorbisenc2-dev/libvorbis-dev/' \
+		-e 's/libwoff2*-dev/libwoff-dev/' \
 		-e 's/libX11-6-dev/libx11-dev/' \
 		-e 's/libX\([a-z]\+\)[0-9]-dev/libx\1-dev/' \
 		-e 's/libxml2-2-dev/libxml2-dev/' \
-- 
2.27.0.rc2

